ios/sensor/ReanimatedSensor.m

```diff
@@ -138,7 +142,7 @@ - (bool)initializeOrientation
   [_motionManager setDeviceMotionUpdateInterval:_interval];
 
   [_motionManager setShowsDeviceMovementDisplay:YES];
-  [_motionManager startDeviceMotionUpdatesUsingReferenceFrame:CMAttitudeReferenceFrameXArbitraryZVertical
+  [_motionManager startDeviceMotionUpdatesUsingReferenceFrame:(1 << _referenceFrame)
```

Comment:
`(1 << _referenceFrame)` looks quite hacky, can you please explain it better in the comment?